pkg/network/protocols/http/model.go

```diff
@@ -27,3 +27,7 @@ type Transaction interface {
 	SetResponseLastSeen(ls uint64)
 	RequestStarted() uint64
 }
+
+func isEncrypted(tx Transaction) bool {
+	return (tx.StaticTags() & (GnuTLS | OpenSSL | TLS)) > 0
```

Comment:
what about gotls and java tls?

Reply:
It was not clear to me that they refer to _encrypted_ traffic because they're currently exported as `Java` and `Go`

Reply:
Done!